FreeRTOS/FreeRTOS-Kernel
# Add macro guard configUSE_MPU_WRAPPERS_V1 to remove definition missing warning.

Description
-----------
This PR adds the guard `configUSE_MPU_WRAPPERS_V1` to the function `prvGetRegionAccessPermissions` to remove warning . 

Test Steps
-----------
NA.

Checklist:
----------



Related Issue
-----------
#1205 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
